### DISCUSSION TITLE
2.3.30 Remove SearchNode Fails to update Grid prior to reboot

### DISCUSSION THREAD

**Discussion by darth-drew** [Security-Onion-Solutions/securityonion#3312] [Unsupported Versions]
Somewhat self explanatory.

Followed this guide:
https://docs.securityonion.net/en/2.3/removing-a-node.html#remove-search-node

ID stills shows offline on the grid page after log out/in. Successful after manager reboot.

**Comment by keepwn** [comment]
Not sure if bug or feature, but the offline node (search) is gone after exec `so-soc-restart` on manager-node.
And, other refrence or file also should be removed:
- /opt/so/saltstack/local/pillar/data/nodestab.sls (refrence in content)
- /opt/so/conf/grafana/grafana_dashboards/search_nodes/$hostname.json
